parser/object_parse/compilation_unit_parse/mod.rs

```diff
@@ -1,10 +1,12 @@
+#[cfg(feature = "dwarf")]
```

Comment:
It could be easier to define a macro like:

```rust
macro_rules! if_dwarf {
    ( $( $x:item )* ) => {
        $(
            #[cfg(feature = "dwarf")]
            $x
        )*
    }
}

if_dwarf! {
    // ... definitions and things for when dwarf is enabled go in here ...
}
```

and then you could use it throughout the rest of the crate rather than adding the `#[cfg(...)]`s manually on every single definition.